src/App.tsx

```diff
@@ -1,14 +1,71 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { FilterCriteria, Select, Status } from './types/otherTypes';
+
+const filterTodos = (
+  todos: Todo[],
+  { status, searchInput }: FilterCriteria,
+): Todo[] => {
+  return todos.filter(todo => {
+    if (status === 'active' && todo.completed) {
+      return false;
+    }
+
+    if (status === 'completed' && !todo.completed) {
```

Comment:
Use a enum for 'active' and 'completed' here
<img width="303" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/64750958/de0602e5-e3cb-45e8-8b58-271cc43b7244">

Reply:
Still not fixed 
<img width="907" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/64750958/499bd0e9-6441-44b3-9146-b5d087edc9dc">